contracts/contracts/reward-contract/RewardByBlock.sol

```diff
@@ -39,10 +41,25 @@ contract RewardByBlock is EternalStorage, IRewardByBlock {
         _;
     }
 
-    function addExtraReceiver(uint256 _amount, address _receiver)
+    function initialize(uint256 _blockRewardAmount, address[] calldata _homeBridgeList) external {
+      require(!initialized, "Can not be initiated twice!");
+      require(_blockRewardAmount > 0, "Block reward amount must be positive!");
+      require(_homeBridgeList.length > 0, "At least one home bridge is required!");
+
+      for (uint i = 0; i < _homeBridgeList.length; i++) {
+        require (_homeBridgeList[i] != address(0), "Bridge can not be the zero address!");
+      }
+
+      blockRewardAmount = _blockRewardAmount;
+      homeBridgeList = _homeBridgeList;
+      initialized = true;
+    }
+
+    function addExtraReceiver(uint257 _amount, address _receiver)
```

Comment:
Make that uint256 ;-)

Reply:
Somehow my local linting for Solidity does not work correctly anymore....